std/experimental/testing/io.d

```diff
@@ -0,0 +1,312 @@
+/**
+ * IO related functions
+ */
+
+module std.experimental.testing.io;
```

Comment:
So I finally took a look at `std.experimental.logger`. With @Dicebot's comment, I feel like the only thing to change is to remove `writelnUt` (I realised later it's pretty much useless, it has the same effect as using plain old `writeln` in a unit test). Or I could remove everything except the ANSI colours from the `io.d` file and completely rework the current IO to use `std.experimental.logger`. The only verbose IO supported by this library was when the `-d` option was used, and all that did was stop turning off stdio and stderr.